README.md

```diff
@@ -18,8 +18,11 @@ These instructions use [homebrew](https://brew.sh/).
 ### Python and Node
 
 1. brew install python3
-1. brew install gunicorn
+1. pip install gunicorn
 1. brew install node@17
```

Comment:
It seems like brew doesn't have node 17. I will likely add a different way to manage node versions here to make it easier